test.sh

```diff
@@ -23,6 +24,16 @@ for dir in ${EXTRA_MOUNT:-}; do
   engine_mounts=("${engine_mounts[@]}" -v "$dir":"$dir":z)
 done
 
+# Force recreation of the container
```

Comment:
test.sh changes are also unrelated to feature, can you extract it into separate commit?

Reply:
By "separate commit", do you mean a different Pull Request? Or a separate commit within this Pull Request?